packages/console/src/consts/connectors.ts

```diff
@@ -20,3 +25,19 @@ export const connectorPlatformLabel: ConnectorPlatformLabel = Object.freeze({
   [ConnectorPlatform.Universal]: 'connectors.platform.universal',
   [ConnectorPlatform.Web]: 'connectors.platform.web',
 });
+
+type ConnectorPlaceholderIcon = {
+  [key in ConnectorType]: SvgComponent;
+};
+
+export const lightModeConnectorPlaceholderIcon: ConnectorPlaceholderIcon = Object.freeze({
+  [ConnectorType.SMS]: SmsConnectorIcon,
+  [ConnectorType.Email]: EmailConnector,
+  [ConnectorType.Social]: EmailConnector, // We don't need a icon for social right now.
```

Comment:
we may need in case the social connector has a broken link

Reply:
You can still return `null` or `undefined` actually

Reply:
> we may need in case the social connector has a broken link

Per offline discussion, we don't need the default social connector icon. Maybe we can add an issue if we want.

Reply:
updated with returning `undefined`